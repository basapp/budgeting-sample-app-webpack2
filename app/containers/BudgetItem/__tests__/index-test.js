import React from 'react';
import renderer from 'react-test-renderer';
import { BudgetItem } from '../index';

// mock nested components
jest.mock('components/DonutChart/Path');
jest.mock('components/BudgetItemDetail', () => 'div');

it('renders correctly', () => {
  const tree = renderer.create(<BudgetItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
