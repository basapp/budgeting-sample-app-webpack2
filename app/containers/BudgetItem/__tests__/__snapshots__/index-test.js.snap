// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  category=""
  percent={0}
  redirectToBudget={[Function]}
  transaction={Object {}}
/>
`;
